api/nodes/serializers.py

```diff
@@ -375,3 +377,61 @@ class Meta:
     @staticmethod
     def get_id(obj):
         return '{}:{}'.format(obj.node._id, obj.provider)
+
+class NodeAlternativeCitationSerializer(JSONAPISerializer):
+
+    id = IDField(source="_id", read_only=True)
+    type = TypeField()
+    name = ser.CharField()
+    text = ser.CharField()
```

Comment:
I'm thinking `name` and `text` should be required fields. That way you won't have to do that check in `error_checker` before creating an `AlternativeCitation` object, which does require the `name` and `text`.